### DISCUSSION TITLE
free-sort plugin no longer working on Mac OS?

### DISCUSSION THREAD

**Discussion by rpomper** [jspsych/jsPsych#2752] [Q&A]
I previously used the free-sort plugin (jsPsych-6.1.0) to create a task that worked on both Mac and Windows computers.

I am attempting to recreate this task using the latest version of jsPsych (7.3.0) and the free-sort plugin no longer works on Mac (I tested in both Safari and Firefox), but does work on Windows. The demo code on the free-sort plug in page also does not work on Mac, but does work on Windows. 

In the console I'm getting the following error:
> Uncaught ReferenceError: TouchEvent is not defined
    value https://unpkg.com/@jspsych/plugin-free-sort@1.0.1:1
    plugin-free-sort$1.0.1:1:6538

Is anyone else able to reproduce this error on Mac? or have an idea what might be causing the error? thanks in advance for any help

![Screen Shot 2022-08-24 at 3 24 56 PM](https://user-images.githubusercontent.com/53839701/186506891-3d5e4d49-9ab2-4e48-8ec0-0886490064a7.png)

**Comment by jodeleeuw** [comment]
Thanks for reporting this. I'm going to open an issue to track it.

**Reply by jodeleeuw** [reply]
#2757
